vcr/pe/presentation_definition_test.go

```diff
@@ -270,51 +307,87 @@ func TestMatch(t *testing.T) {
 }
 
 func Test_matchFormat(t *testing.T) {
-	verifiableCredential := credential.ValidNutsOrganizationCredential(t)
-
 	t.Run("no format", func(t *testing.T) {
 		match := matchFormat(nil, vc.VerifiableCredential{})
 
 		assert.True(t, match)
 	})
 
 	t.Run("countable format", func(t *testing.T) {
-		match := matchFormat(&PresentationDefinitionClaimFormatDesignations{}, vc.VerifiableCredential{})
+		match := matchFormat(&PresentationDefinitionClaimFormatDesignations{}, credentialToJSONLD(vc.VerifiableCredential{}))
 
 		assert.True(t, match)
 	})
 
-	t.Run("format with jwt_vc always returns false", func(t *testing.T) {
-		asMap := map[string]map[string][]string{"jwt_vc": {"alg": {"ES256K", "ES384"}}}
-		asFormat := PresentationDefinitionClaimFormatDesignations(asMap)
-		match := matchFormat(&asFormat, vc.VerifiableCredential{})
+	t.Run("JWT", func(t *testing.T) {
+		const jwtCredential = `eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImRpZDpleGFtcGxlOmFiZmUxM2Y3MTIxMjA0
```

Comment:
You might want to prevent people from using this example at other points because it's no longer valid (expired). Or generate a valid credential for each test run?

Reply:
I used it because it came from the VC data model, so we at least have some tests that use data from the specification. I'll add a comment that it's expired and where it came from.